### PR TITLE
Add message app

### PR DESCRIPTION
**Pull request by TomerNewmanPrograms**
Fixing #31 
Dependent on PR #25 , please do not merge it  before #25 

-Adding a new message app including the models, tests, views, init, admin and apps python files

### CODE REVIEW

File: ResuMe/settings.py [hunk 1]
```diff
@@ -37,7 +37,7 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
-    'resumeApp.apps.ResumeappConfig',
+    'Feed.apps.FeedConfig',
```
Comment (OmerCS8) [inline ResuMe/settings.py:40]: Maybe add here `messages.apps.MessagesConfig` .

**Review by OmerCS8 — CHANGES_REQUESTED** [review]

**Review by OmerCS8 — APPROVED** [review]
LGTM

**Review by MatanP12 — APPROVED** [review]
LGTM

**Review by yulisuliman — APPROVED** [review]
LGTM

File: Feed/views.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+from django.shortcuts import render
+
+
+def Feed(request):
```
Comment (AlonShakaroff) [inline Feed/views.py:4]: The method name should start with a lower case letter, I know you base on my PR and I changed it so maybe edit those changes out of the PR or fix it according to the name standards of methods like Nati mentioned in my PR.

**Review by AlonShakaroff — CHANGES_REQUESTED** [review]
I think the changes to resumeApp name to Feed shouldn't be part of this PR.
In any case, I changed my original PR that you based on, so that the method called Feed is changed to feed to fit the PEP8 coding standards.

File: Feed/views.py (hunk 2 above)
Comment (TomerNewmanPrograms) [reply Feed/views.py:4]: Yes I'm waiting for your new changes to rebase my PR according to yours

**Review by abiton1 — APPROVED** [review]

**Pull request closed without merging**